views/update/overseas-entity-query.html

```diff
@@ -12,15 +12,30 @@
 {% endblock %}
 
 {% block content %}
```

Comment:
Is the sign out link working in this page?

Reply:
There is no sign out link on this page

Reply:
We have a bug for that: https://companieshouse.atlassian.net/browse/UAR-216